Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -768,33 +754,72 @@ void GameListCtrl::RescanList()
   std::set_difference(search_results.cbegin(), search_results.cend(), cached_paths.cbegin(),
                       cached_paths.cend(), std::back_inserter(new_paths));
 
-  const Core::TitleDatabase title_database;
+  // Reload the TitleDatabase
+  m_title_database = {};
```

Comment:
I didn't think of it before, but... Can't this lead to thread safety issues if disc drives are being listed?